### ISSUE TITLE
Target C*Os on home page

### ISSUE THREAD

**Issue by stevepiercy** [Pylons/trypyramid.com#102]
## tl;dr
We need content that conveys the message, "Hey, C*O! Here's why Pyramid is the perfect fit for you!"

## Back story
Originally with @pauleveritt, @blaflamme, myself, and I forget who all else, we wanted to address multiple target audiences with the trypyramid.com marketing website, including CFO/CTOs and other non-programmer decision-makers, and new and experienced developers. In another effort from @goodwillcoding and Gavin(?), the one-file app from the Pyramid docs was duplicated and placed front and center with instructions of how to install Pyramid via `pip`, and that was all. As a result of that merge of concepts, we left out content and a presentation that effectively conveys the message to CFO/CTOs that Pyramid is perfect for them.

There have been a few other changes since then, and the site continues to be developed and evolve.

I don't have examples of how to do this. I'm not a C*O of a big company, and I don't know what they seek in a web application framework, much less how to address it.

**Comment by stevepiercy** [comment]
I found the perfect way to convey the message to C*Os

![yousa like pyramid](http://i.imgur.com/KRNE9P7.jpg)

**Comment by Dowwie** [comment]
hoping this helps, too

https://i.imgflip.com/13b5pi.jpg

**Comment by Dowwie** [comment]
hoping this helps.. 
![](https://i.imgflip.com/13b5pi.jpg)

**Comment by Dowwie** [comment]
Hopefully this helps:

In financial services (the "wall street" inv banks), I started in IT as a "front office" programmer but added on to that role product development project-lead kind of roles and also got into risk management related work.  Then I spent some time in b-school at Duke but left to start up a venture.  When I realized that *I* would have to be the one to create the first iteration of my web applications (poor me, I know), I researched my options and arrived at Pyramid being the best fit.  The way that I decided on this is as follows:

## Pyramid is Designed for Growth
Pyramid is a framework that lets its users decide what modules to use with it.  I get to control how many important parts function, and I like having that flexibility.  Flexibility is important to me for a number of reasons.  I anticipate changes in the architecture/design of the system as my product evolves.  Further, as more experienced web app developers join the project, they can improve parts without needing to replace the whole (if I did it alright).  

## Pyramid is trusted technology
Pyramid has been thoroughly battle tested.  It is based on a rich history of trial and error.  When I adopt Pyramid, I'm adopting a design based on tough lessons learned along the way with many iterations of improvement.  I trust the technology more because of that.

## Pyramid offers Optimal Performance
Pyramid offers "optimal performance", which I'm defining as strong performance with  flexibility.  I don't want a bare metal solution.  Bare metal would force me to confront many of the challenges that others did throughout the pyramid legacy.  Instead, Pyramid is a lean mean Mr.T fighting machine, offering the best features at the best performance -- the muscle without the fat.
